src/components/pages/ParentHome/ParentNavbar.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Menu, Layout } from 'antd';
+import 'antd/dist/antd.css';
+import {
+  ClockCircleOutlined,
+  GiftOutlined,
+  FundProjectionScreenOutlined,
+  CarryOutOutlined,
+} from '@ant-design/icons';
+import { useOktaAuth } from '@okta/okta-react';
+
+const { Sider } = Layout;
```

Comment:
Does this get used? And same question about authService and onCollapse variables.